### PR TITLE
Fix the search bar bug

### PR DESCRIPTION
**Pull request by lubosmj**
[noissue]

ref: https://github.com/pulp/pulpcore/pull/3668

### CODE REVIEW

File: templates/docs/docs/conf.py.j2 [hunk 1]
```diff
@@ -30,7 +30,9 @@ except ImportError:
 
 # Add any Sphinx extension module names here, as strings. They can be extensions
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
-extensions = ['sphinx.ext.extlinks']
+extensions = ['sphinx.ext.extlinks', 'sphinx.ext.autodoc', 'sphinx.ext.autosummary',
```
Comment (ggainey) [inline templates/docs/docs/conf.py.j2:33]: Do these need to all be called out in doc_requirements as well?

**Comment by ipanova** [comment]
@lubosmj these are templates. They will not be updated, they will only be generated once during plugin bootstrapping.

**Comment by ipanova** [comment]
Also, i believe not every plugin uses `sphinx-rtd-theme`, or at least we're not enforcing that.

**Comment by lubosmj** [comment]
Seems like there are more problems than with the original issue. You are both right about declaring unnecessary requirements.

If the template will not be updated, there is no need to continue working on this PR.

**Comment by ggainey** [comment]
> If the template will not be updated, there is no need to continue working on this PR.

Aye, you're right. Apologies for suggesting you do something here for this - good thing we have @ipanova to keep us on track :)

**Pull request closed without merging**